release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -568,6 +564,9 @@ The `ImageContentSourcePolicy` (ICSP) object is now deprecated. You can now use
 For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
 
 For more information on converting existing ICSP YAML files to IDMS YAML files, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
+[id="ocp-4-13-toolbox-deprecation"]
+==== Toolbox is deprecated in {op-system}
+The toolbox library is deprecated and support will be removed from a future {product-title} release.
```

Comment:
from a future {product-title} release, or in a future {product-title} release? 

There are other instances of this (the RHV note, and the Wildcard note) , but this is my first time actually encountering it.